src/scss/partials/_logo.scss

```diff
@@ -31,39 +31,29 @@ $logo-color--transparent    : $white;
         }
     }
 }
-    .c-logo-img img, .c-logo-img svg {
-        // default logo image height and width (as should be an inline SVG)
-        width: 82px;
-        height: 16px;
 
-        @include media('>=mid') {
-            width: 165px;
-            height: 32px;
-        }
+.c-logo-img img, .c-logo-img svg {
+    // default logo image height and width (as should be an inline SVG)
+    width: 82px;
+    height: 16px;
 
-         // Menulog logo, as it has multiple fill values built in. We just hide the outline on transparent mode.
-        @if ($theme == 'ml') {
-            width: 118px;
-            height: 26px;
-            margin-left: -10.5px; //half of hamburger menu width
+    @include media('>=mid') {
+        width: 165px;
+        height: 32px;
+    }
 
+      // Menulog logo, as it has multiple fill values built in. We just hide the outline on transparent mode.
+    @if ($theme == 'ml') {
+        width: 92px;
```

Comment:
Should this not be `120 x 32` on mobile, and then below `149 x 41`?

Reply:
I was going by what's on Zeplin, please could you show me where those dimensions are from?